test/test_admin.py

```diff
@@ -1,12 +1,18 @@
+from __future__ import annotations
+
```

Comment:
This shouldn't be necessary?

Reply:
For type annotations without quotes, it is necessary. I can add quotes.